### ISSUE TITLE
Solarized color accessibility

### ISSUE THREAD

**Issue by chharvey** [altercation/OLD-ethanschoonover.com-original#32]
I noticed that while checking foreground colors on top of background colors, some combinations do not meet the WCAG 2.0 accessibility specifications; namely, that [the contrast ratio must be at least 4.5](https://www.w3.org/TR/WCAG/#visual-audio-contrast).

Here's a great [tool that calculates the contrast ratio](http://leaverou.github.io/contrast-ratio/) between foreground and background colors.

For example, Solarized Base 00 color on a Solarized Base 3 background (the default setting for light mode) yields a contrast ratio of 3.4, which does not meet requirements unless the text is large or bold. Further, Solarized Base 1 (light mode comments) on the same background yields a contrast ratio of 2.5, which fails all requirements.

On a personal note, I use the Solarized Dark scheme for my command line, with Base 03 as the background. But instead of the recommended Base 0 for text, I used Base 1 (emphasis) for normal text because Base 0 is too hard to read (even though the contrast ratio is 4.7).

I realize that completely addressing this accessibility issue is probably not going to give you the dynamic range that you need to make a beautiful color palette, but I was wondering if you could mention the WCAG specs on [your website](http://ethanschoonover.com/solarized#usage-development)? If nothing more than to just let other developers know it's being acknowledged.

If I have made an error or am mistaken, please kindly correct me. Thank you.

**Comment by chharvey** [comment]
<img width="725" alt="screen shot 2016-01-30 at 01 37 27" src="https://cloud.githubusercontent.com/assets/1362083/13708619/a227f346-e77d-11e5-85c1-8f79fd1b8d58.png">

**Comment by nortexoid** [comment]
I recently noticed the same thing and tweaked my theme to have a contrast of 4.99:1 which is what you get with base 3 background and base 01 foreground (text). The scheme could use tweaking to improve contrast.
